app/javascript/dashboard/components/widgets/EmptyState.vue

```diff
@@ -2,7 +2,7 @@
   <div class="empty-state py-16 px-1 ml-0 mr-0">
     <h3
       v-if="title"
-      class="text-slate-700 dark:text-slate-200 block text-center w-full text-4xl font-thin"
+      class="text-slate-700 dark:text-slate-200 block text-center w-full text-xl font-thin"
```

Comment:
Felt this to be a bit bigger than it should be. 

| Before | After |
| -- | -- |
| <img width="1512" alt="Screenshot 2024-03-03 at 7 51 34 PM" src="https://github.com/chatwoot/chatwoot/assets/2246121/e55055fe-fd56-42a9-a621-cdf306508a23"> | <img width="1508" alt="Screenshot 2024-03-03 at 7 51 20 PM" src="https://github.com/chatwoot/chatwoot/assets/2246121/86d4e482-df31-4568-a0e3-468e2addc946"> |